pkg/tfbridge/diff.go

```diff
@@ -70,7 +70,11 @@ func visitPropertyValue(name, path string, v resource.PropertyValue, tfs *schema
 	rawNames bool, visitor propertyVisitor) {
 
 	if IsMaxItemsOne(tfs, ps) {
-		v = resource.NewArrayProperty([]resource.PropertyValue{v})
+		if v.IsNull() {
+			v = resource.NewArrayProperty([]resource.PropertyValue{})
+		} else {
+			v = resource.NewArrayProperty([]resource.PropertyValue{v})
+		}
```

Comment:
Can we cover this with test coverage?

Reply:
Yup, I'll go ahead and do that before merging